scrapers/base.py

```diff
@@ -46,3 +47,28 @@ def run(self, since=None):
         :raises NotImplementedError: if the function is not overridden
         """
         raise NotImplementedError()
+
+    def isContainerBuild(self, attached_build):
+        """
+        Check whether a build is a container build.
+
+        :param KojiBuild attached_build: A build attached to an advisory in question
+        :return: boolean value indicating whether the builf is a container build
+        :rtype: bool
+        """
+        package_name = attached_build['package_name']
+        try:
+            extra_json = json.loads(attached_build['extra'])
+        except (ValueError, TypeError):
+            extra_json = {}
+
+        # Checking a heuristic for determining if a build is a container build since, currently
+        # there is no definitive way to do it.
+        if extra_json and extra_json.get('container_koji_build_id'):
```

Comment:
This is missing a heuristic I added in https://github.com/release-engineering/estuary-api/commit/b07ab726aa135b9b7af49f6a34a60b61222acd0d